src/Common/nunit/ColorConsole.cs

```diff
@@ -76,7 +76,6 @@ public static ConsoleColor GetColor(ColorStyle style)
                             return ConsoleColor.DarkGray;
                         case ColorStyle.Help:
                             return ConsoleColor.DarkGray;
-                        case ColorStyle.Default:
                         default:
```

Comment:
Looks like a personal preference here but one that differs from the author. Not part of our standards.